aladdin.sh

```diff
@@ -280,9 +280,7 @@ function enter_docker_container() {
         -e "HOST_ADDR=$HOST_ADDR" \
         -e "command=$command" \
         `# Mount host credentials` \
-        `# Mount destination for aws creds will not be /root/.aws because we will possibly make` \
-        `# changes there that we don't want propagated on the host's ~/.aws` \
-        -v "$(pathnorm ~/.aws):/root/tmp/.aws" \
+        -v "$(pathnorm ~/.aws):/root/.aws" \
```

Comment:
no longer making changes to the ~/.aws files, so don't need to mount it to a unique location